### PR TITLE
Remove Delete Folder button when root is selected [OSF-6139]

### PR DESCRIPTION
**Pull request by barbour-em**
## Purpose

"Delete Folder" button should not be available when root node/folder of a provider is selected.

## Changes

- Hide delete folder if the root is selected.
- Update the docs for setting up GH because the settings page changed some.

## Side effects

Nothing, clicking delete on the root didn't do anything before. This just removes the button.

## Ticket

https://openscience.atlassian.net/browse/OSF-6139
[OSF-6139]

### CODE REVIEW

File: website/addons/github/static/githubFangornConfig.js [hunk 1]
```diff
@@ -171,15 +171,16 @@ var _githubItemButtons = {
...
+                        }, 'Create Folder')
                     );
+                    if(!item.data.isAddonRoot){
+                        buttons.push(m.component(Fangorn.Components.button, {
+                            onclick: function (event) {
+                                _removeEvent.call(tb, event, [item]);
+                            },
+                            icon: 'fa fa-trash',
+                            className: 'text-danger'
+                        }, 'Delete Folder'));
```
Comment (samanehsan) [inline website/addons/github/static/githubFangornConfig.js]: This if statement is missing a closing `}`

**Comment by samanehsan** [comment]
Pass complete @barbour-em! ☀️

**Comment by barbour-em** [comment]
Back to you @samanehsan! 💯

**Comment by samanehsan** [comment]
Looks good to me @barbour-em! 🐙

**Pull request merged**